InterviewService.swift

```diff
@@ -0,0 +1,71 @@
+
+import Foundation
+
+enum Mode {
+    case preprod
+    case prod
+}
+
+enum TypeService {
+    case Network
+    case appSetting
+    case mock
+    case calculate
+}
+
+enum Flow {
+    case firstFlow, secondFlow, thirdFlow
+}
+
+protocol NewServiceInput {
+    func getType() -> Any?
+    func calculateValue()
+}
+
+final class NewService: NSObject {
+    var typeService: TypeService
+    var mode: Mode
+    
+    //Get data from database
```

Comment:
Do not leave comments for simple actions.

Comments for public methods and entities are welcome.